### ISSUE TITLE
IANA WebSocket Protocol Registry has broken link to WAMP specification

### ISSUE THREAD

**Issue by max-cura** [wamp-proto/wamp-proto#402]
https://www.iana.org/assignments/websocket/websocket.xml lists http://wamp.ws/spec as the WAMP specification URL, but said link is currently broken (it currently redirects to https://wamp-proto.org/spec, which serves an XML error file with code "NoSuchKey").

**Comment by KSDaemon** [comment]
Hi @max-cura Thanks for pointing out! 
If you know who can correct it to https://wamp-proto.org/spec.html will be cool. Otherwise we'll try to fix it on our owns.

**Comment by max-cura** [comment]
IANA listed @oberstet as the 'Reference' for the protocol, which after looking through some RFCs seems to indicate that they're the point-of-contact for that protocol?

Otherwise my best guess would just be shoot an email to the official IANA address.

**Comment by KSDaemon** [comment]
@oberstet Hey, Tobias! Can you fix it in iana docs?

**Comment by oberstet** [comment]
yeah, long ago, but I registered `wamp` early on in 2012. anyways, looking at the registry infos, I can see these errors rgd WAMP:

1. Link

The link is broken, it should point to https://wamp-proto.org/spec.html 

2. Name

rather than simply `wamp` (which was used for v1 of WAMP), implementations actually use `wamp.2.json` and generally the subprotocol name template `wamp.2.<serializer>.[batched]`, thus, the whole name-prefix `wamp` should be understood to be registered / claimed - I don't know if the IANA number registry and IETF RFC6455 whould even allow for that.

> practically, above hasn't been a problem. at least not to my awareness ..

3. Contact

The contact is wrong

> [Tobias_Oberstein]	Tobias Oberstein	mailto:tobias.oberstein&tavendo.de	2012-07-10
